pkg/network/usm/ebpf_javatls.go

```diff
@@ -164,6 +164,14 @@ func newJavaTLSProgram(c *config.Config) *JavaTLSProgram {
 	}
 }
 
+func (p *JavaTLSProgram) Name() string {
+	return "java-tls"
+}
+
+func (p *JavaTLSProgram) IsBuildModeSupported(buildMode) bool {
```

Comment:
should be similar to go (or add support for prebuilt for java, as it shouldn't be difficult)

Reply:
java already supports prebuilt